tree/ntuple/v7/test/ntuple_type_name.cxx

```diff
@@ -70,6 +70,7 @@ TEST(RNTuple, TClassDefaultTemplateParameter)
       model->AddField(RFieldBase::Create("f4", "struct DataVector<bool,vector<unsigned>>").Unwrap());
       model->AddField(RFieldBase::Create("f5", "DataVector<Double32_t>").Unwrap());
       model->AddField(RFieldBase::Create("f6", "DataVector<int, double>").Unwrap());
+      model->MakeField<DataVector<StructUsingCollectionProxy<int>>>("f7");
```

Comment:
I am still not finding a dictionary for `DataVector<StructUsingCollectionProxy<int>>` ... am I missing something or is that intentional?

Reply:
Missed that again, apologies :)